13 спринт

IvanPodlesnykh opened the pull request.

IvanPodlesnykh committed:
13 Спринт
1. Сверстал экран плеера
2. Реализовал переход на экран плеера при нажатии на трек
3. Реализовал перенос данных о треке в активити плеера

IvanPodlesnykh committed:
13 Спринт
1. Исправил размер обложки в плеере
2. Ограничил размер названия трека, артиста и альбома

nasibullin1412 commented:
⚠️На устройствах с небольшим экраном вёрстка экрана «Аудиоплеер» едет (эмулятор Nexus 5x)

![image](https://github.com/IvanPodlesnykh/PlaylistMaker/assets/57040048/ea016594-d12e-4625-baa3-ae3b720b21a2)

nasibullin1412 commented on app/src/main/java/com/ivanpodlesnykh/playlistmaker/PlayerActivity.kt:
```diff
@@ -0,0 +1,82 @@
+        }
+    }
+
+    private fun handleTrackInfo() {
```
🍏Слишком большой метод, советую разбить на несколько поменьше

nasibullin1412 commented on app/src/main/res/layout/activity_player.xml:
```diff
@@ -0,0 +1,212 @@
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@+id/player_back_button"
+        app:layout_constraintStart_toEndOf="@id/left_guideline"
+        app:layout_constraintEnd_toStartOf="@+id/right_guideline"
```
🍏Вёрстка едет, скорей всего, из-за того, что не определён атрибут Bottom_toTopOf. Постарайся на этом экране для каждой вью аккуратно его прописать

IvanPodlesnykh committed:
13 Спринт
Исправил некорректное отображение на небольших экранах
Добавил ограничения для всех View по вертикальной цепочке + размер обложки изменяется по отношению к размеру экрана
